spec/services/ncr/approval_manager_spec.rb

```diff
@@ -175,6 +175,26 @@
       end
     end
 
+    context "for a BA60 or BA61 request" do
+      it "uses BA61 tier1 team approver when org code matches" do
+        org_letters = %w( 7 J 4 T 1 A C Z )
+        org_letters.each do |org_letter|
+          org_code = "P11#{org_letter}XXXX"
+          ncr_org = create(:ncr_organization, code: org_code)
+          ba60_work_order = create(:ba60_ncr_work_order, ncr_organization: ncr_org)
+          ba61_work_order = create(:ba61_ncr_work_order, ncr_organization: ncr_org)
+          ba80_work_order = create(:ba80_ncr_work_order, ncr_organization: ncr_org)
+          ba60_work_order.setup_approvals_and_observers
+          ba61_work_order.setup_approvals_and_observers
+          ba80_work_order.setup_approvals_and_observers
+
+          expect(ba60_work_order.budget_approvals.first.user_id).to eq(Ncr::Mailboxes.ba61_tier1_budgetteam.id)
```

Comment:
yes I expected that to work too, and I started with that. But there is some wonkiness is comparing the AR objects there and I got an error (it was not comparing them using the User pk).